tools/generated_files/400/supportedExprs.csv

```diff
@@ -1,845 +1,846 @@
-Expression,Supported,SQL Func,Notes,Context,Params,BOOLEAN,BYTE,SHORT,INT,LONG,FLOAT,DOUBLE,DATE,TIMESTAMP,STRING,DECIMAL,NULL,BINARY,CALENDAR,ARRAY,MAP,STRUCT,UDT,DAYTIME,YEARMONTH
```

Comment:
All lines changed to add a new column: OBJECT

Now:
320+:
override def getAllSupportedTypes(): TypeEnum.ValueSet = TypeEnum.values - TypeEnum.OBJECT

400+:
override def getAllSupportedTypes(): TypeEnum.ValueSet = TypeEnum.values

Seems we need to remove the OBJECT type.